open-source-terms.md

```diff
@@ -149,37 +148,67 @@ may have to use Paid Services, are subject to the following conditions:
-    expect other users of npm Services to unwittingly execute that
-    malicious code.
-
-12. You will not falsely imply that you are affiliated with or endorsed
+11. You will not falsely imply that you are affiliated with or endorsed
     by npm.
 
-13. You will not operate illegal schemes, such as pyramid schemes, via
+12. You will not operate illegal schemes, such as pyramid schemes, via
     npm Services.
 
-14. You will not deep-hyperlink to images or other non-hypertext content
+13. You will not deep-hyperlink to images or other non-hypertext content
     served by npm Services.
 
-15. You will not remove any marking indicating proprietary ownership
+14. You will not remove any marking indicating proprietary ownership
     from any material got via npm Services.
 
-16. You will not display any portion of the Website via an HTML IFRAME.
+15. You will not display any portion of the Website via an HTML IFRAME.
 
-17. You will not disable, avoid, or circumvent any security or access
+16. You will not disable, avoid, or circumvent any security or access
     restrictions of npm Services, or access parts of npm Services not
     intended for access by you.
 
-18. You will not strain infrastructure of npm Services with an
+17. You will not strain infrastructure of npm Services with an
     unreasonable volume of requests, or requests designed to impose an
     unreasonable load on IT systems underlying npm Services.
 
-19. You will not encourage or assist any other person in violation of
+18. You will not encourage or assist any other person in violation of
     "Acceptable Use".
 
+## Acceptable Content
+
+Administrators at npm reserve the right to delete content hosted on
+the npm Services that is deemed unacceptable. Unacceptable content
+can take the form of a package, a README file, a user or organization
+name, or any other content submitted to the npm Services. Content
+that is unacceptable includes but is not limited to:
+
+1. Content that is illegal, offensive, or otherwise harmful. This includes
+   content that is harassing, inappropriate or abusive.
+
+2. Content in violation of law, infringing the intellectual property
+   rights of others, violating the privacy or other rights of others,
+   or in violation of any agreement with a third party. This includes
+   code which violates a license.
+
+3. Content containing malicious computer code, such as computer viruses,
+   computer worms, rootkits, back doors, adware, or spyware. This
+   includes content submitted for research purposes unless agreed to
+   in advance by npm. Tools designed and documented explicitly to
+   assist in security research are acceptable, but proof-of-concept
+   exploits are not.
+
+4. Packages that are not functionally compatible with the npm
+   command-line client. For example, a "package" cannot be a PNG or
+   JPEG image, a movie file, or a text document. Using the Public
+   Registry as a general purpose database is not allowed.
+
```

Comment:
Kat and I have been discussing dropping support for those from the cli, though she may have changed her mind now with npx. =p